src/components/atoms/CheckBox/CheckBox.mdx

```diff
@@ -0,0 +1,23 @@
+---
+name: Check Box
+route: /CheckBox
+---
+
+import { Playground, Props } from 'docz';
+import CheckBox from './CheckBox';
```

Comment:
Typically "Checkbox" is one word... Please change if possible

Please note, making commits where you only change capitalization are tricky though, because in OSX capitalization doesn't matter but in Unix it does, so it will cause weird problems when you push.

Make sure you set one intermediate commit (ie, `CheckBox.tsx -> CheckBoxTemp.tsx --> Checkbox.tsx`)